resources/core/charts/k8s-dashboard-proxy/templates/deployment.yaml

```diff
@@ -0,0 +1,105 @@
+        app: {{ template "name" . }}
+        release: {{ .Release.Name }}
+    spec:
+      serviceAccount:  {{ template "name" . }}
+      {{ if .Values.global.isLocalEnv }}
+      hostNetwork: true
+      # dnsPolicy: ClusterFirstWithHostNet
+      {{ end }}
+      containers:
+      - image: {{ .Values.keycloak.image }}
+        name: {{ template "name" . }}
+        {{ if not .Values.global.isLocalEnv }}
+        resources:
+          limits:
+            memory: 256Mi
+          requests:
+            memory: 128Mi
+        {{ end }}
+        imagePullPolicy: IfNotPresent
+        args:
+          - --discovery-url=https://dex.{{ .Values.global.domainName }}
+          - --skip-openid-provider-tls-verify=true
+          - --client-id=kubernetes-dashboard
+          - --client-secret=oiEWUWOIEwedfgg
+          - --resources=uri=/*|methods=GET,POST,PUT,DELETE
+          - --redirection-url=https://kubernetes-dashboard.{{ .Values.global.domainName }}
+          - --verbose=true
+          - --listen=0.0.0.0:{{ .Values.containerPort }}
+          - --upstream-url=http://{{ .Values.reverseproxy.host }}:{{ .Values.reverseproxy.port }}
+          - --enable-session-cookies=true
+          - --enable-logging=true
+          - --secure-cookie=true
+          - --enable-authorization-cookies=false
+          - --enable-token-header=false
+          - --enable-encrypted-token=true
+          - --enable-security-filter=true
+          - --filter-browser-xss=true
+          - --filter-content-nosniff=true
+          - --filter-frame-deny=true
+          - --encryption-key=AgXa7xRcoClDEU0ZDSH4X0XhL5Qy2Z2j
+          - --http-only-cookie=true
+          - --upstream-keepalives=true
+          - --upstream-keepalive-timeout=60s
+          - --enable-logout-redirect=true
+          - --oauth-uri=/oauth
+          - --cookie-domain=https://kubernetes-dashboard.{{ .Values.global.domainName }}
+          - --cookie-access-name=KYMA_TOKEN
+        ports:
+        - containerPort: {{ .Values.containerPort}}
+        livenessProbe:
+          httpGet:
+            path: /oauth/health
+            port: {{ .Values.containerPort}}
+          initialDelaySeconds: 100
+          timeoutSeconds: 20
+        readinessProbe:
+          httpGet:
+            path: /oauth/health
+            port: {{ .Values.containerPort}}
+          initialDelaySeconds: 30
+          timeoutSeconds: 20
+        volumeMounts:
+        - mountPath: /var/run/secrets/tokens
+          name: svc-account-token
+      # - image: {{ .Values.global.containerRegistry.path }}/{{ .Values.global.k8s_dashboard_proxy.dir }}k8s_dashboard_proxy:{{ .Values.global.k8s_dashboard_proxy.version }}
+      - image: k8s-dashboard-proxy:{{ .Values.global.k8s_dashboard_proxy.version }}
+        name: reverse-proxy
+        imagePullPolicy: IfNotPresent
+      volumes:
+      - name: svc-account-token
+        projected:
+          sources:
+          - serviceAccountToken:
+#            path: svc-account-token
```

Comment:
nit: please cleanup commented fragments

Reply:
You're right definitely, this was left for improvement step to read the secret value from a volume. Will remove if I didn't manage to make it as a part of this PR ✌️